pkg/kubelet/rkt/rkt.go

```diff
@@ -181,6 +181,7 @@ type Runtime struct {
 }
 
 var _ kubecontainer.Runtime = &Runtime{}
+var _ kubecontainer.DirectStreamingRuntime = &Runtime{}
```

Comment:
Have a question: is there any initialization problem of assigning `kubecontainer.DirectStreamingRuntime` in both dockertools and rkt package?